test/Microsoft.ML.Benchmarks/Harness/CIBenchmark.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.ML.Benchmarks.Harness
+{
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+    public class CIBenchmark : Attribute
```

Comment:
It might be a good idea to add a `<summary>` comment to this attribute explaining its purpose.